.github/workflows/CI.yml

```
@@ -0,0 +1,95 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+
+env:
+  __TUACMDR_PROGRESS_BARS__: "disabled"
```

This appears to be a typo.

```suggestion
  __TAUCMDR_PROGRESS_BARS__: "disabled"
```